test/rpc/eth/getBlockByHash.js

```diff
@@ -1,176 +1,81 @@
 const test = require('tape')
 
-const request = require('supertest')
 const { INVALID_PARAMS } = require('../../../lib/rpc/error-code')
-const { startRPC, closeRPC, createManager, createNode } = require('../helpers')
+const { baseSetup, params, baseRequest } = require('../helpers')
 const { checkError } = require('../util')
 
-test('call eth_getBlockByHash with valid arguments', t => {
-  const manager = createManager(createNode())
-  const server = startRPC(manager.getMethods())
+let method = 'eth_getBlockByHash'
```

Comment:
`method` is not reassigned so it should be `const`

Reply:
Right. Will address in an update. @ryanio @tomonari-t thanks for the feedback, will the directly sit down and apply to the other files.